plugins/runtime_id_plugin/runtime_id_plugin.c

```diff
@@ -0,0 +1,36 @@
+#include "runtime_id_plugin.h"
+
+#include <sodium.h>
+
+#include "uuid/uuid.h"
+
+datadog_php_uuid runtime_id = DATADOG_PHP_UUID_INIT;
+
+typedef enum datadog_php_plugin_result plugin_result_t;
+
+static plugin_result_t datadog_php_runtime_id_plugin_minit(int type, int module_number) {
+    (void)type;
+    (void)module_number;
```

Comment:
Do you think there is any value in passing these args in at all?

Reply:
I believe `type` is TEMPORARY or PERMANENT -- we definitely need that, don't we?

Reply:
Yeah, `MODULE_TEMPORARY` means the module has been loaded with `dl()` which we don't support anyway. So it might actually make sense to add a check for this in the main MINIT similar to the check we do to [test the observer API](https://github.com/php/php-src/blob/803779e/ext/zend_test/test.c#L388-L396) and fail early. Then we can ignore the args altogether. What do you think?